presto-docs/src/main/sphinx/sql/alter-view.rst

```diff
@@ -0,0 +1,42 @@
+============
```

Comment:
```suggestion
==========
```
Formatting lines should be the same number of characters as the text being formatted.